code/modules/mob/living/carbon/superior_animal/roach/types/roachling.dm

```diff
@@ -9,7 +9,9 @@
 	move_to_delay = 3.5
 
 	melee_damage_lower = 2
-	melee_damage_upper = 5
+	melee_damage_upper = 3
+
+	waring_faction_multy = 3 //We are young, so much to live for
```

Comment:
This makes no sense, a roachling shouldn't be able to do more damage than a Kraftwork or a common roach.

Reply:
6-9 when attacking a spider, when you corner something wild with its life to live for.
Thats the idea

Reply:
Right but why would a BABY roach deal more damage than an ADULT roach when cornered?

Reply:
Big roaches have had a life, they are older, less to live and love for. They are past their prime!